packages/x-date-pickers/src/internals/hooks/useField/useFieldState.ts

```diff
@@ -61,6 +65,7 @@ export const useFieldState = <
     internalProps: {
       value: valueProp,
       defaultValue,
+      referenceDate: referenceDateProp,
```

Comment:
Is there a reason for this renaming? I do not see other `referenceDate` in the code


```suggestion
      referenceDate,
```

Reply:
It was named `referenceValue` before, thus the renaming
But not now that it's `referenceDate` I can remove the renaming :+1:

Reply:
My bad, we already have a `referenceDate` in `parseDateStr`